### PR TITLE
Fix/inspector flex container unset labels

### PR DESCRIPTION
**Pull request by enidemi**
**Problem:**
Some inspector control labels are shown as 'set' while their values are actually empty.

**Fix:**
Fixing align items, flex wrap rows.

**Commit Details:**
- wrapped align items in contextmenu wrapper that applies the color change on hover
- pass controlstatus to flex wrap menu

### CODE REVIEW

File: editor/src/components/inspector/controls/option-chain-control.tsx [hunk 1]
```diff
@@ -45,6 +45,7 @@ export const OptionChainControl: React.FunctionComponent<DEPRECATEDControlProps<
       display: 'flex',
       flexDirection: 'column',
       marginBottom: 0,
+      width: '100%',
```
Comment (Rheeseyb) [inline editor/src/components/inspector/controls/option-chain-control.tsx:48]: Were these changes intended as part of this?

**Review by Rheeseyb — APPROVED** [review]

File: editor/src/components/inspector/controls/option-chain-control.tsx (hunk 1 above)
Comment (enidemi) [reply editor/src/components/inspector/controls/option-chain-control.tsx:48]: yes, the align-items control is now in a contextmenu wrapper and it shrinked in size.
